compiler/include/misc.h

```diff
@@ -23,7 +23,12 @@
 #include <cstdio>
 #include <cstdlib>
 
+#ifdef HAVE_LLVM
+#define exit(x) clean_exit(x)
+#else
+// This interferes with uses of exit() in LLVM header files.
 #define exit(x) dont_use_exit_use_clean_exit_instead
+#endif
```

Comment:
I think this is OK for now but note that there is a better solution that amounts to extending the work done in PR #7233 - that is, extend util/devel/lookForBalloc to also check for bad exits.

Reply:
Oh, nice catch.  I didn't realize we were doing that.  It looks like the current place is util/devel/look_for_calls.py, which is already looking for exit!

What I'll do is commit this as is, but post-commit:

@ronawho - Can we remove this entirely now that we sanitize the code of calls to exit?
```c
#define exit(x) dont_use_exit_use_clean_exit_instead
```

Reply:
We only check the runtime for calls to exit. See [util/devel/lookForBadRTCalls](https://github.com/chapel-lang/chapel/blob/master/util/devel/lookForBadRTCalls) or the PR Michael pointed out. You're welcome to beef up the checks to include the compiler too, but you'll either want to rename the script or create a new `checkForBadCompCalls` or something